hedera-node/hedera-app/src/main/java/com/hedera/node/app/grpc/impl/netty/NettyGrpcServerManager.java

```diff
@@ -133,9 +154,28 @@ public NettyGrpcServerManager(
                             builder.query(m.path());
                         }
                     });
-                    return builder.build(metrics);
+                    return builder.build(metrics, true);
                 })
                 .collect(Collectors.toUnmodifiableSet());
+
+        final var grpcConfig = configProvider.getConfiguration().getConfigData(GrpcConfig.class);
+        if (grpcConfig.nodeOperatorPortEnabled()) {
+            // Convert the various RPC service definitions into query endpoints permitting unpaid queries for node
+            // operators
+            nodeOperatorServices = rpcServiceDefinitions
+                    .get()
```

Comment:
Could extract this chain to a private method that accepts an extra filter on what methods to register or something.

Reply:
Updated.